runtime/server/queries_metrics.go

```diff
@@ -127,14 +127,14 @@ func (s *Server) metricsQuery(ctx context.Context, instanceId string, priority i
 	return structTypeToMetricsViewColumn(rows.Schema), data, nil
 }
 
-func buildMetricsTopListSql(req *runtimev1.MetricsViewToplistRequest, mv *runtimev1.MetricsView) (string, []any, error) {
+func buildMetricsTopListSQL(req *runtimev1.MetricsViewToplistRequest, mv *runtimev1.MetricsView) (string, []any, error) {
 	dimName := quoteName(req.DimensionName)
 	selectCols := []string{dimName}
 	for _, n := range req.MeasureNames {
 		found := false
 		for _, m := range mv.Measures {
 			if m.Name == n {
-				expr := fmt.Sprintf(`%s as "%s"`, m.Expression, m.Name)
+				expr := fmt.Sprintf(`%s as %q`, m.Expression, m.Name)
```

Comment:
Revert – this is SQL formatting, not Go formatting, so this can be dangerous if Go changes behavior.